### PR TITLE
Implement 'snapshots [snapshotID]'

### PR DESCRIPTION
**Pull request by baurmatt**

### CODE REVIEW

File: restic/internal/snapshots.py [hunk 1]
```diff
@@ -3,9 +3,12 @@
 from restic.internal import command_executor
 
 
-def run(restic_base_command, group_by=None):
+def run(restic_base_command, snapshot_id=None, group_by=None):
     cmd = restic_base_command + ['snapshots']
 
+    if snapshot_id:
+        cmd.extend([snapshot_id])
```
Comment (mtlynch) [inline restic/internal/snapshots.py:10]: It's a little simpler to just do `cmd.append(snapshot_id)`.

**Review by mtlynch — CHANGES_REQUESTED** [review]
Looks good! One minor note.

File: restic/internal/snapshots.py (hunk 1 above)
Comment (baurmatt) [reply restic/internal/snapshots.py]: Thanks, fixed! :)

**Review by mtlynch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**